cmd/network/verification.go

```diff
@@ -257,17 +272,23 @@ func (e *EgressVerification) Run(ctx context.Context) {
 		// Prompt sending a service log instead for other blocked egresses.
 		if !out.IsSuccessful() && len(out.GetEgressURLFailures()) > 0 {
 			failures++
-			postCmd := generateServiceLog(out, e.ClusterId)
-			blockedUrl := strings.Join(postCmd.TemplateParams, ",")
-			if (strings.Contains(blockedUrl, "deadmanssnitch") || strings.Contains(blockedUrl, "pagerduty")) && e.cluster.State() == "ready" {
-				fmt.Println("PagerDuty and/or DMS outgoing traffic is blocked, resulting in a loss of observability. As a result, Red Hat can no longer guarantee SLAs and the cluster should be put in limited support")
-				pCmd := lsupport.Post{Template: limitedSupportTemplate}
-				if err := pCmd.Run(e.ClusterId); err != nil {
-					fmt.Printf("failed to post limited support reason: %v", err)
+
+			// Only send service logs if not disabled by flag
+			if !e.NoServiceLog {
```

Comment:
Could this be `if e.NoServiceLog { fmt.Printlin; return }` so we don't need to nest the main code?